.github/workflows/tests.yml

```diff
@@ -13,51 +13,56 @@ concurrency:
 permissions:
   contents: read
 
+env:
+  DJANGO_SETTINGS_MODULE: tracdjangoplugin.settings_tests
+  PYTHONWARNINGS: >
+    error,
+    default:::passlib.utils,
```

Comment:
I may be uninitiated, but I don't understand why this is being included here.

Reply:
I added a comment to try and explain the presence of `default:::passlib.utils` (which I got to by trial and error)